gps/turn/index.html

```diff
@@ -21,20 +21,21 @@ <h2 class="text-left"><i class="fas fa-fw fa-info-circle"></i> About this Networ
 <p>Eighteen <abbr title="National Geodetic Survey">NGS</abbr> Continually Operating Reference Stations (CORS) were selected along the perimeter and interior of the network and were constrained to the published coordinates to conduct the final adjustment of the network coordinates.</p>
 <p>With an activated subscription, set your GPS unit's VRS connection address to <code>168.179.231.9</code> and port <code>2101</code>. Enter your new device-specific username and password.</p>
 <h5>Connections from Source Table</h5>
-<p>For Trimble units we have three different connections.</p>
+<p>For Trimble units we have these different connections.</p>
```

Comment:
Is three any different than these?

Reply:
It may change later down the road, wanted to eliminate a number.